Client/qtTeamTalk/keycomp.ui

```diff
@@ -35,7 +35,7 @@
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
-         <string>Hold down the keys which should be used as a hot key. Modifier keys like Shift, Ctrl and Alt can be used in combination with other keys</string>
+         <string>Hold down the keys that you want to used as a hot key. Modifier keys like Shift, Ctrl and Alt can be used in combination with other keys</string>
```

Comment:
This correction is minimal and will invalidate all existing translations. I would prefer to stay with the original